### PR TITLE
Clearer chanjo-report error message only when needed

### LINKED ISSUE #4675: Import chanjo-report warning emitted before log level set
When running CLI commands in an environment where chanjo-report is not correctly installed - which could be for good reason, maybe it is not even configured in that instance - a warning is issued as an import exception is handled. This warning is emitted before colored logs have been installed, and looks both ominous and pretty odd. Let's give the CLI users a neater experience!

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
- Closes #4675. Don't try to import the chanjo-report lib unless `SQLALCHEMY_DATABASE_URI` is present in settings

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
### When you have SQLALCHEMY_DATABASE_URI in settings but not chanjo-report lib installed:

image

### When you don't have SQLALCHEMY_DATABASE_URI set in settings and chanjo-report is NOT installed:

image

File: scout/server/extensions/chanjo_extension.py [hunk 1]
```diff
@@ -9,21 +9,22 @@
...
     def init_app(self, app):
+        try:
+            from chanjo_report.server.app import configure_template_filters
+            from chanjo_report.server.blueprints import report_bp
+            from chanjo_report.server.extensions import api as chanjo_api
+        except ImportError as error:
+            chanjo_api = None
+            report_bp = None
+            configure_template_filters = None
+            LOG.error(f"{error}")
```
Comment (dnil) [inline scout/server/extensions/chanjo_extension.py:26]: The format string and variable passing is in general the preferred way to delay instantiation, but I guess no-one is going to run this at a log level below error. I can live with the eye-sore. 😁

**Review by dnil — APPROVED** [review]
Yes, excellent!

File: scout/server/extensions/chanjo_extension.py (hunk 1 above)
Comment (northwestwitch) [reply scout/server/extensions/chanjo_extension.py:26]: Ah it was a longer message before, I'll remove the format string thing.

**Pull request merged**